### ISSUE TITLE
Onsen Click-Events (when used on form with ng-submit) will fail since iOS 8.4.1

### ISSUE THREAD

**Issue by Hirbod** [OnsenUI/OnsenUI#858]
Apple has changed synthtic clicks - most ng-submits on my forms fail to work.
This happens due fastclick.js - We need an immediate update to fastclick 1.0.6

https://github.com/ftlabs/fastclick/issues/421

My App does not work anymore properly

```
WebKit
Available for: iPhone 4s and later, iPod touch (5th generation) and later, iPad 2 and later
Impact: A malicious website can make a tap event produce a synthetic click on another page
Description: An issue existed in how synthetic clicks are generated from tap events that could cause clicks to target other pages. The issue was addressed through restricted click propagation.
CVE-ID
CVE-2015-5759 : Phillip Moon and Matt Weston of Sandfield
```

**Comment by Hirbod** [comment]
@argelius I can confirm fastclick 1.0.6 will fix the issues. Please update als soon aus possible

**Comment by Hirbod** [comment]
seemed to be fixed in 1.3.8, I'm on an older version currently.

**Comment by argelius** [comment]
@Hirbod Sorry for the late reply. I have been on vacation the last two weeks.

I ran some tests with iOS 8.4 without any issues so I will close this.
